po/template.pot

```diff
@@ -258,6 +91,9 @@ msgstr ""
 msgid "Import"
 msgstr ""
 
+msgid "Cancel"
```

Comment:
It seems there is some new translations. Are .po files ready too? I can't see them in this merge.

Reply:
There shouldn't be any new translations, but entries template.pot have changed order during `create-translations`